### PR TITLE
Expose focusNode

### PR DESCRIPTION
**Pull request by leonardo2204**

### CODE REVIEW

**Comment by leonardo2204** [comment]
@jpnurmi Hi there, thanks for this great project!
See if this PR is ok, please!

Thanjs

**Review by jpnurmi — COMMENTED** [review]
Thank you, looks good. 👍 Just one minor comment about an import that slipped in possibly by accident?

File: lib/src/cupertino/spin_box.dart [hunk 1]
```diff
@@ -21,6 +21,7 @@
 // SOFTWARE.
 
 import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
```
[the remark this answers is not in the document]
Comment (leonardo2204) [reply lib/src/cupertino/spin_box.dart]: Ops, you are totally right, missed that one!
Just fixed it, thanks for pointing that out!

**Review by jpnurmi — APPROVED** [review]
The CI failure is unrelated to this. Something has changed in the beta channel, which the CI was still accidentally using ever since the null-safety migration.

**Comment by jpnurmi** [comment]
Thanks again. v0.7.0 was published: https://pub.dev/packages/flutter_spinbox/versions/0.7.0

**Pull request merged**